sdg_mapping/sdg_index/sdg_index_utils.py

```diff
@@ -59,16 +66,66 @@ def parse_2019_sdg_indicator(dataset):
     trend_map = maps['trend_map']
     achievement_map = maps['achievement_map']
 
-    sdg_index_19_df = dataset
-    trend_columns = [i for i in sdg_index_19_df.columns if 'Trend' in i]
-    dashboard_columns = [i for i in sdg_index_19_df.columns if (('Dashboard' in i) and ('Goal' in i))]
+    trend_columns = [i for i in df.columns if 'Trend' in i]
+    dashboard_columns = [i for i in df.columns if (('Dash' in i) and ('Goal' in i))]
+
+    for col in trend_columns:
+        df[col] = df[col].map(trend_map)
+    for col in dashboard_columns:
+        df[col] = df[col].map(achievement_map)
+
+    return df
+
+
+def _parse_raw(df, year):
+    """_parse_raw
+    Currently returns raw data as is.
+    """
+    return df
+
+
+def _parse_report(df, year):
+    """_parse_report
+    Parse data and map values for the SDG Index report worksbook sheet.
+
+    Args:
+        df (pd.DataFrame): Raw SDG Index report table.
+        year (int): Year of the index. 
+
+    Returns:
+        df (pd.DataFrame): Clean data.
+    """
+    df = _map_dashboard_symbols(df)
+    df.columns = [c.lower().replace(" ", "_") for c in df.columns]
+    if year == 2017:
+        old = "average_score_on_sdg"
+        new = "goal_"
+        df.columns = [c.replace(old, new) + "_score" for c in df.columns]

```

Comment:
Is the underscore in `goal_` needed?

Reply:
Yeah. The originals are like `average_score_on_sdg11` and to be consistent with other years the result needs to be `goal_11_score`

Reply:
Oh right, that makes sense.